docs/Makefile

```diff
@@ -21,18 +36,18 @@ livehtml:
 .PHONY: help livehtml Makefile
 
 
-env/Miniconda3-latest-Linux-x86_64.sh:
+env/Miniconda3:
 	mkdir env
-	wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O env/Miniconda3-latest-Linux-x86_64.sh
-	chmod a+x env/Miniconda3-latest-Linux-x86_64.sh
+	wget https://repo.anaconda.com/miniconda/Miniconda3-latest-$(OSFLAG)-x86_64.sh -O env/Miniconda3-latest-$(OSFLAG)-x86_64.sh
+	chmod a+x env/Miniconda3-latest-$(OSFLAG)-x86_64.sh
 
 env:
 	rm -rf env
-	make env/Miniconda3-latest-Linux-x86_64.sh
-	./env/Miniconda3-latest-Linux-x86_64.sh -p $(PWD)/env -b -f
-	source env/bin/activate; conda config --system --add envs_dirs $(PWD)/env/envs
-	source env/bin/activate; conda config --system --add pkgs_dirs $(PWD)/env/pkgs
-	source env/bin/activate; conda env update --name base --file ./environment.yml
+	make env/Miniconda3
+	./env/Miniconda3-latest-$(OSFLAG)-x86_64.sh -p $(PWD)/env -b -f
+	. env/bin/activate; conda config --system --add envs_dirs $(PWD)/env/envs
```

Comment:
Why the change from `source` to `.`?

Reply:
Using `source` might cause this to fail on some systems (it failed on my Ubuntu 18.04) because `make` runs on the `sh` shell, that does not support `source`. It would only work for those that have `sh` being actually the `dash` shell installed in the system (this is what I found online).